.travis.yml

```diff
@@ -1,6 +1,9 @@
 language: ruby
 rvm:
   - 2.2.3
+before_script:
+  - sed -f _scripts/baseurl.sed _config.yml > _scripts/_config.yml
+  - mv _scripts/_config.yml _config.yml 
```

Comment:
To Michael's other comment about whether to do this inline or have a sed file, I agree that doing it inline is probably better.  I think you can remove the "mv" line and replace the sed line with:

```sed -ie '/baseurl/s/\/m-lab\.github\.io//' _config.yml```

The -i option make the change on the file in-place, so no need to redirect the output.  And adding /baseurl/ to the expression assures us that this operation will _only_ happen on the line containing "baseurl", just in case we ever, for some reason, have another entry or line with the same string (unlikely, but good form).